pkg/chaosdaemon/util/util.go

```diff
@@ -22,11 +22,17 @@ import (
 	"strconv"
 	"sync"
 
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/graph"
+
 	"github.com/pkg/errors"
 
 	"github.com/chaos-mesh/chaos-mesh/pkg/bpm"
 )
 
+var log = ctrl.Log.WithName("chaos-daemon/util")
```

Comment:
@STRRL how about adding the naming convention of logger to logging rfc?

Reply:
I aggree with you ,fixed.

Reply:
> @STRRL how about adding the naming convention of logger to logging rfc?

+1 

I would raise another pr about  logger naming convention

Reply:
> > @STRRL how about adding the naming convention of logger to logging rfc?
> 
> +1
> 
> I would raise another pr about logger naming convention

Updated in https://github.com/chaos-mesh/rfcs/pull/37